### ISSUE TITLE
CrystalFormat has not triggered for some split window

### ISSUE THREAD

**Issue by sudo-nice** [rhysd/vim-crystal#48]
Hello, thank you for the plugin!

Steps to reproduce the issue:

1. Create two files, say,

a.cr:

```
def foo
  "foo"
end
```

  b.cr:
```
def bar
  "bar"
end
```
2. Open the files with split windows:
`[n]vim -o a.cr b.cr`

3. Make some changes to file `a.cr` (e.g. change lines indentation) and save the buffer: `:w`.

*Expected result:* the CrystalFormat should be triggered and lines should be reindented.
*Actual result:* No format changes happened.

But for the file `b.cr` all works just fine.

I've tried Nvim 0.2.2 with this minimal config:
```
" initialize
call plug#begin('~/.config/nvim/plugged')

Plug 'rhysd/vim-crystal'
let g:crystal_auto_format = 1

" finalize
call plug#end()
```

**Comment by rhysd** [comment]
Yes, auto formatting is only for the current buffer in order to avoid formatting an unexpected file. So the file in another window will not be formatted.

**Comment by rhysd** [comment]
@sudo-nice Thank you for reporting this. I fixed this at d1acf59. Please update to the latest.

**Comment by sudo-nice** [comment]
Thank you!
